spring-cloud-dataflow-shell/src/main/resources/application.yml

```diff
@@ -6,3 +6,9 @@ spring:
 logging:
   pattern:
     console:
+  file:
```

Comment:
> Add commented shell log file settings which is a way to log hard shell startup errors

**Q1**: What is the behavior w/o this file (is this something new in 3.2.2)?

**Q2**: If we ship this w/ debug logging, how does a user override this?

Reply:
Hold on, this was meant to left there as commented out, just like in skipper.